src/server/main.py

```diff
@@ -298,8 +298,6 @@ def refresh(unique_id: str) -> Response:
     except SqlError as e:
         return make_response({"message": f"Database connection error: {e}"}, 500)
 
-    user_email: str = ""
```

Comment:
Since we remove the line with typehint, can we add it on the line where `user_email` is created?